packages/core/src/discovery.js

```diff
@@ -132,7 +133,6 @@ async function* captureSnapshotResources(page, snapshot, options) {
     if (captureWidths) options = { ...options, width };
     let captured = await page.snapshot(options);
     captured.resources.delete(normalizeURL(captured.url));
-    capture(processSnapshotResources(captured));
```

Comment:
is L134 required?

Reply:
Yes, we set it to `baseSnapshot` from which we eventually extract the page HTML

Reply:
Sorry I meant L135